cdtb/__main__.py

```diff
@@ -327,6 +327,8 @@ def command_bin_dump(parser, args):
     if args.json:
         json_dump(binfile.to_serializable(), sys.stdout)
     else:
+        if binfile.linked_files is not None:
+            print(f"linked: {binfile.linked_files}")
```

Comment:
The format should be closer to the syntax used by entries and their fields. And files should be listed one per line (more readable and easier to grep).

Something like this:
```
<Linked [
  file1
  file2
  ...
]>
```
Implementation is easy since we can reuse helpers from `binfile.py`:
```py
slinked = _repr_indent_list(binfile.linked_files)
print(f"<Linked {slinked}>")
```
Moreover, it could be a good idea to move the printing code to a new `BinFile` method.